pontoon/base/static/js/translate.js

```diff
@@ -1649,6 +1650,10 @@ var Pontoon = (function (my) {
      */
     updateProfileMenu: function () {
       $('#profile .admin-current-project a').attr('href', '/admin/projects/' + this.project.slug + '/');
+
+      $('#profile .download .resource')
+        .html(this.entities[0].format.toUpperCase())
+        .toggle(!this.project.win);
```

Comment:
A reference to the iframe window where in-place localization takes place. We use this to check if the website supports in-place localization. Because if it does, it might be linked to several resources, not just one, so instead of downloading the resource, we offer you to download HTML. Which is probably not very useful. In the future, we should probably allow you to download a ZIP of all resources (but it will take time to generate for e.g Firefox).